server/routes/officeRoutes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import OfficeController from '../controllers/OfficeController';
```

Comment:
I exported the router module from "const router = express.Router()",
I OfficeController from my controller file so that i can have access to the function createOffice i created there

Reply:
Check the casing of the OfficeController from where you import it. Since it is an instance of the class, the casing should be camelCase.

Reply:
ok let me do that immediately